torch/headeronly/core/Dispatch.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <torch/headeronly/core/ScalarType.h>
+
+#define AT_PRIVATE_CASE_TYPE_USING_HINT_TMPL(PRELUDE, enum_type, HINT, ...)   \
```

Comment:
```suggestion
#define THO_PRIVATE_CASE_TYPE_USING_HINT_TMPL(PRELUDE, enum_type, HINT, ...)   \
```
and below too for clarity